packaging/linux/run_keybase

```diff
@@ -118,6 +118,11 @@ forward_vars() {
   done
 }
 
+run_redirector() {
+  logdir="${XDG_CACHE_HOME:-$HOME/.cache}/keybase"
+  nohup keybase-redirector /keybase >> $logdir/keybase.redirector.log 2>&1 &
```

Comment:
Any reason why doesn't this log to the same directory as `run_redirector` in the post-install script? Also since this is the root redirector for all users, it's seems better to keep it in a dir owned by root.

Reply:
Because the user starting the service can't write to a log file in a directory owned by root.  We'd need to use sudo for that, and we want to avoid sudo.

Reply:
Oh that's right. Can we have the root redirector write to the log file directly?

Reply:
Also it seems there's a way to allow user to start a system service: https://unix.stackexchange.com/questions/192706/how-could-we-allow-non-root-users-to-control-a-system-d-service

Reply:
The root redirector currently doesn't do any logging, this was just to capture any errors that are printed back to the terminal.  Anyway, the Rust redirector has some other story for logging so it's not really worth any time right now I think.

Reply:
Is there any worry that (if there's any output at all) this file will grow without bounds? I don't imagine anyone is ever going to rotate it like we do with the client/kbfs logfiles? (Though I guess the GUI logfile has a similar problem for now.)

Reply:
Growing without bounds is always a possibility of course, but I think we should strive to avoid any output from this binary at all, except for errors.  So it's not something I'm worried about in practice.